Themis/Views/Assessment/CorrectionSidebar/ProblemStatementCellView.swift

```diff
@@ -20,9 +20,35 @@ struct ProblemStatementCellView: View {
             Text("Problem Statement")
                 .font(.largeTitle)
 
+            ZStack {
+                Divider()
+                    .frame(height: 5)
+                    .overlay(.gray)
+                HStack {
+                    ForEach(vm.bundledTests, id: \.testCase) { bundledTest in
+                        if bundledTest != vm.bundledTests.first {
+                            Spacer()
+                        }
+                        if bundledTest.passed {
+                            Image("TestPassedSymbol")
```

Comment:
I would strongly recommend creating the icons as a custom view considering how simple it is. This would save a fair amount of memory (both icons use 600mb of storage).

Reply:
600mb?